libnmstate/netapplier.py

```diff
@@ -129,8 +127,9 @@ def _apply_ifaces_state(desired_state, verify_change, commit,
 
 
 def _verify_change(desired_state):
-    current_state = state.State({Constants.INTERFACES: netinfo.interfaces()})
+    current_state = state.State(netinfo.show())
```

Comment:
Another unrelated change, is it not?

Reply:
This is related as State now holds the route configuration also.

Reply:
This is a good reason. Would be great to mention these kind of reasons in the commit message - it could have been a separate commit as well in preparation for the route changes.